### PR TITLE
JIRA1597: changes instances of moactl and -moa- to rosa.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
This PR is for MOA/ROSA issue https://issues.redhat.com/browse/OSDOCS-1597

Link to manual docs build: http://file.rdu.redhat.com/jonwilli/moa/JIRA1597/welcome/index.html

Peer reviewers: Changes were made to all of the assemblies under Getting Started. You can run a search for "moactl" to make sure it's changed to "rosa". Also, the examples that contain -moa- in strings should now all be changed to -rosa-

No QE review is needed, so if this passes peer review, please go ahead and merge to [enterprise-4](https://issues.redhat.com/browse/enterprise-4).5 branch. Thanks!

### CODE REVIEW

File: modules/moa-accessing-your-cluster.adoc [hunk 1]
```diff
@@ -55,21 +55,21 @@ Any optional fields can be left empty and a default will be selected.
...
 The IDP can take 1-2 minutes to be configured within your cluster.
 .. Enter the following command to verify that your IDP has been configured correctly:
 +
 [source,terminal]
 ----
-$ moactl list idps --cluster rh-moa-test-cluster1
+$ rosa list idps --cluster rh-rosa-test-cluster1
 ----
 +
 .Example output
```
Comment (jeana-redhat) [inline modules/moa-accessing-your-cluster.adoc:75]: line 79 has a `rh-moa-test` in it

File: modules/moa-accessing-your-cluster.adoc [hunk 2]
```diff
@@ -93,8 +93,8 @@ $ moactl describe cluster rh-moa-test-cluster1
 Name:        rh-moa-test-cluster1
```
Comment (jeana-redhat) [inline modules/moa-accessing-your-cluster.adoc:93]: `rh-moa-test`

File: modules/moa-accessing-your-cluster.adoc [hunk 3]
```diff
@@ -108,13 +108,13 @@ Created:     May 27, 2020
...
 ----
-$ oc login --token=z3sgOGVDk0k4vbqo_wFqBQQTnT-nA-nQLb8XEmWnw4X --server=https://api.rh-moa-test-cluster1.j9n4.s1.devshift.org:6443
+$ oc login --token=z3sgOGVDk0k4vbqo_wFqBQQTnT-nA-nQLb8XEmWnw4X --server=https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443
 ----
 +
 .Example output
 [source,terminal]
 ----
-Logged into "https://api.rh-moa-test-cluster1.j9n4.s1.devshift.org:6443" as "rh-moa-test-user" using the token provided.
+Logged into "https://api.rh-moa-test-cluster1.j9n4.s1.devshift.org:6443" as "rh-rosa-test-user" using the token provided.
```
Comment (jeana-redhat) [inline modules/moa-accessing-your-cluster.adoc:117]: `rh-moa-test`

File: modules/moa-creating-cluster.adoc [hunk 4]
```diff
@@ -47,17 +47,17 @@ Enter the following command to check the status of your cluster. During cluster
...
 ----
 
 .Example output
 [source,terminal]
 ----
-Name:        rh-moa-test-cluster1
+Name:        rh-rosa-test-cluster1
 ID:          1de87g7c30g75qechgh7l5b2bha6r04e
 External ID: 34322be7-b2a7-45c2-af39-2c684ce624e1
 API URL:     https://api.rh-moa-test-cluster1.j9n4.s1.devshift.org:6443
```
Comment (jeana-redhat) [inline modules/moa-creating-cluster.adoc:59]: `rh-moa-test`

File: modules/moa-installing-prerequisites.adoc [hunk 5]
```diff
@@ -83,7 +83,7 @@ $ moactl
 Command line tool for MOA.
```
Comment (jeana-redhat) [inline modules/moa-installing-prerequisites.adoc:83]: MOA -> ROSA in this context right?

File: modules/moa-installing-prerequisites.adoc [hunk 6]
```diff
@@ -161,11 +161,11 @@ After both the permissions and quota checks pass, proceed to preparing your AWS
...
-.. Run the following command to log in to your Red Hat account with `moactl`.
+.. Run the following command to log in to your Red Hat account with `rosa`.
 +
 [source,terminal]
 ----
-$ moactl login
+$ rosa login
 ----
 +
 Replace `<my_offline_access_token>` with your token.
```
Comment (jeana-redhat) [inline modules/moa-installing-prerequisites.adoc:171]: In line 176 below, the URL https://cloud.redhat.com/openshift/token/moa still works but redirects to https://cloud.redhat.com/openshift/token/rosa. May want to update.

File: modules/moa-installing-prerequisites.adoc [hunk 7]
```diff
@@ -192,7 +192,7 @@ I: Logged in as 'rh-moa-user' on 'https://api.openshift.com'
 +
```
Comment (jeana-redhat) [inline modules/moa-installing-prerequisites.adoc:192]: Line 188 above has `rh-moa-user`

File: modules/moa-installing-prerequisites.adoc [hunk 8]
```diff
@@ -215,7 +215,7 @@ OCM Organization External ID: 0000000
...
 [source,terminal]
 ----
-$ moactl init
+$ rosa init
 ----
 +
 .Example output
```
Comment (jeana-redhat) [inline modules/moa-installing-prerequisites.adoc:221]: Line 224 below has `rh-moa-user`

**Comment by jonquilwilliams** [comment]
@jeana-redhat Sorry for the strays. After talking with Kevin, it was ok to use the Find All/Replace All in Atom to change the file names, directories, etc. So everything has changed to rosa now. Updated manual preview is here:

http://file.rdu.redhat.com/jonwilli/moa/JIRA1597/welcome/

You can ignore moa in this path because it's just my local naming until I get around to changing that. No impact on this JIRA card. Thank you!

**Review by jeana-redhat — COMMENTED** [review]
Found a typo, but otherwise LGTM! Good work, I know my eyes got crossed just checking it lol

File: web_console/web-console.adoc [hunk 9]
```diff
@@ -22,6 +22,6 @@ infrastructure for your cluster.
 include::modules/web-console-overview.adoc[leveloffset=+1]
 endif::openshift-enterprise,openshift-webscale,openshift-origin,openshift-aro[]
 
-ifdef::openshift-moa[]
-To access your web console please refer to the xref:../getting_started_moa/creating-first-moa-cluster.html#moa-accessing-your-cluste    r[accessing your cluster] section in the guide to creating your first cluster.
-endif::openshift-moa[]
+ifdef::openshift-rosa[]
+To access your web console please refer to the xref:../getting_started_rosa/creating-first-rosa-cluster.html#rosa-accessing-your-cluste    r[accessing your cluster] section in the guide to creating your first cluster.
```
[the remark this answers is not in the document]
Comment (jonquilwilliams) [reply web_console/web-console.adoc]: @jeana-redhat Argh - I must have leaned on the keyboard. Fixed. I tested that the link works in the Accessing the Web Console topic. Preview here: http://file.rdu.redhat.com/jonwilli/moa/JIRA1597/welcome/

And thank you!!!!!!!!!!!

**Pull request merged**